apps/common-app/src/examples/RuntimeTests/ReanimatedRuntimeTestsRunner/TestRunner.ts

```diff
@@ -564,6 +569,25 @@ export class TestRunner {
     });
   }
 
+  public waitForAnimationUpdates(snapshot: Array<any>): Promise<boolean> {
+    const updatesCount = snapshot.length;
+    const flag = makeMutable(false);
+    return new Promise<boolean>(resolve => {
+      // eslint-disable-next-line @typescript-eslint/no-misused-promises
+      const interval = setInterval(async () => {
+        await this.runOnUIBlocking(() => {
+          'worklet';
+          assertMockedAnimationTimestamp(global.framesCount);
+          flag.value = global.framesCount >= updatesCount - 1;
```

Comment:
Do we need updatesCount at all? Perhaps it would be enough to wait until the number of recorded frames doesn't increse during 1000/FPS ms? 🤔

Reply:
As we discussed offline, we have decided to abandon this approach